nodes/scheduling/secondary_scheduler/nodes-secondary-scheduler-release-notes.adoc

```diff
@@ -12,6 +12,26 @@ These release notes track the development of the {secondary-scheduler-operator-f
 
 For more information, see xref:../../../nodes/scheduling/secondary_scheduler/index.adoc#nodes-secondary-scheduler-about_nodes-secondary-scheduler-about[About the {secondary-scheduler-operator}].
 
+[id="secondary-scheduler-operator-release-notes-1.3.2_{context}"]
+== Release notes for {secondary-scheduler-operator-full} 1.3.2
+
+Issued: 19 November 2025
+
+The following advisory is available for the {secondary-scheduler-operator-full} 1.3.2:
+
+* link:https://access.redhat.com/errata/RH??-2025:TODO[RH??-2025:TODO]
+// TODO: Update with advisory link on release day
+
+[id="secondary-scheduler-operator-1.3.2-new-features-and-enhancements_{context}"]
+=== New features and enhancements
+
+* This release rebuilds the base image for the {secondary-scheduler-operator} to improve its image grade.
+
+[id="secondary-scheduler-operator-1.3.2-known-issues_{context}"]
+=== Known issues
+
+* Currently, you cannot deploy additional resources, such as config maps, CRDs, or RBAC policies through the {secondary-scheduler-operator}. Any resources other than roles and role bindings that are required by your custom secondary scheduler must be applied externally. (link:https://issues.redhat.com/browse/WRKLDS-645[*WRKLDS-645*])
```

Comment:
I like the wording in the Jira better (we should document that _they are responsible for creating additional resources on their own_). Seems much simpler; I wasn't sure before reading the Jira what _must be applied externally_ means. But, I'm not target audience, though.
```suggestion
* Currently, you cannot deploy additional resources, such as config maps, CRDs, or RBAC policies through the {secondary-scheduler-operator}. You are responsible for creating any resources, other than roles and role bindings, that are required by your custom secondary scheduler. (link:https://issues.redhat.com/browse/WRKLDS-645[*WRKLDS-645*])

 they are responsible for creating additional resources on their own
```